CLAUDE.md

```diff
@@ -0,0 +1,9 @@
+# Coding conventions for terraware-server
+
+Always include a trailing newline when working on mjml, html, and ftl files.
```

Comment:
In addition to having this sentence here, we could also add a Spotless rule for this.

Reply:
Ah I didn't realize that was possible with spotless; I thought it was only useful for jvm-based files. That would be great, I was considering adding it to CONVENTIONS.md

Reply:
If you look at the spotless block in `build.gradle.kts` it should have some examples to crib from.